src/app/registration/registration.component.css

```diff
@@ -2,17 +2,21 @@ a {
   color: #459fed;
   text-decoration: none;
 }
+
 .title1 {
   display: block;
   text-align: center !important;
   font-size: 35px;
 }
+
 .title1 > * {
```

Comment:
.title1 > * { top: 150px} First of, never use px values (% or rem), second, I suppose title1 holds only one element, and it is moved down a bit? What would happen if you would add second element?